lib/src/enums.dart

```diff
@@ -4,31 +4,53 @@
 
 /// Values for the event name to be sent to Google Analytics
 ///
-/// The [label] for each enum value is what will be logged, the [desc]
+/// The [label] for each enum value is what will be logged, the [description]
 /// is here for documentation purposes
 enum DashEvents {
   hotReloadTime(
     label: 'hot_reload_time',
     description: 'Hot reload duration',
-    toolOwner: 'flutter_tools',
+    toolOwner: DashTools.flutterTools,
   ),
   ;
 
   final String label;
   final String description;
-  final String toolOwner;
+  final DashTools toolOwner;
   const DashEvents({
     required this.label,
     required this.description,
     required this.toolOwner,
   });
 }
 
+/// The tools that have been onboarded
+///
+/// All tools should use a hyphen as a delimiter
+enum DashTools {
```

Comment:
The convention is to name enums singular

```suggestion
enum DashTool {
```